person_capture/gui_app.py

```
@@ -7260,6 +7350,24 @@ def save_hit(
+                        "reid_dist": c.get("rd"),
+                        "x1": primary_row_crop[0],
+                        "y1": primary_row_crop[1],
+                        "x2": primary_row_crop[2],
+                        "y2": primary_row_crop[3],
+                        "crop_path": crop_img_path,
+                        "sharpness": c.get("sharp"),
+                        "ratio": str(ratio_str),
+                        "face_fraction": c.get("face_frac"),
+                        "profile": c.get("crop_profile"),
+                        "status": "KEPT",
+                        "span_id": int(span_i) if keep_spans else None,
```

Ruff (0.15.12)

[warning] Function definition does not bind loop variable `span_i`

(B023)